server/src/test/java/org/opensearch/indices/IndicesRequestCacheTests.java

```diff
@@ -368,29 +370,17 @@ public void testCacheCleanupBasedOnZeroThreshold() throws Exception {
         DirectoryReader reader = OpenSearchDirectoryReader.wrap(DirectoryReader.open(writer), new ShardId("foo", "bar", 1));
         TermQueryBuilder termQuery = new TermQueryBuilder("id", "0");
         BytesReference termBytes = XContentHelper.toXContent(termQuery, MediaTypeRegistry.JSON, false);
-        if (randomBoolean()) {
-            writer.flush();
-            IOUtils.close(writer);
-            writer = new IndexWriter(dir, newIndexWriterConfig());
-        }
-        writer.updateDocument(new Term("id", "0"), newDoc(0, "bar"));
```

Comment:
these things were unnecessary for my tests, so removed from all of my tests added previously